ui/app/pages/swaps/swaps-gas-customization-modal/swaps-gas-customization-modal.container.js

```diff
@@ -121,18 +128,20 @@ const mapStateToProps = (state) => {
     customGasLimitMessage,
     customTotalSupplement,
     usdConversionRate: getUSDConversionRate(state),
-    disableSave: insufficientBalance || customGasLimit < minimumGasLimit,
+    disableSave: insufficientBalance || customGasLimitTooLow,
     minimumGasLimit,
```

Comment:
This is a string, yet the PropType for the component expects this to be a number

Reply:
addressed in https://github.com/MetaMask/metamask-extension/pull/9858/commits/d8ab5a05cfcfaf3a100d4ba4249ee938efd96b5d

Reply:
I think we want to change the PropType, not convert the value to a `number`, which may be unsafe.

Reply:
`AdvancedGasInputs` has used a `number` for the gas limit for some time now. I think changing that would take more effort. I'm also not sure we need to deal with anything above the max int size in this case, since we're using GWEI rather than WEI.

Reply:
I see. In that case I'm fine with it.

As a general rule, I'm against using numbers for any Ethereum quantities while our type checking is the way it is, but nothing to do here.